src/Router.jsx

```diff
@@ -29,13 +30,15 @@ const Router = () => {
       ],
     },
     {
-      path: '/teams',
-      element:
-  <RequireAuth>
-    <ConnectedLayout />
-  </RequireAuth>,
+      path: '/',
+      element: (
+        <RequireAuth>
+          <ConnectedLayout />
+        </RequireAuth>
+      ),
       children: [
-        { path: '', element: <ListTeams /> },
+        { path: 'overview', element: <Overview /> },
+        { path: 'teams', element: <ListTeams /> },
         { path: 'user/:teamID', element: <AddUserToTeam /> },
```

Comment:
Hum, maybe we should keep the same pattern from routes as backend ? So /team/:teamId/user ?

Reply:
Yes i agree with your feedback! We can set `/team/:teamID` for AddUserToTeam no?

Reply:
Hum.. maybe we should keep the /team/:teamID to a possible view to display all details of the team, and we have the update route also so maybe /user at the end for add user and /update for the update ?